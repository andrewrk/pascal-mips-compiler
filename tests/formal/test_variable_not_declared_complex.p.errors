ERROR: line 11: variable "undeclaredVariable" not declared
