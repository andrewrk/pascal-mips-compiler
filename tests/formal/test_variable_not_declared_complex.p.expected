ERROR: line 11: Undeclared varable: undeclaredVariable
